app/code/community/Bolt/Boltpay/Model/ShippingAndTax.php

```diff
@@ -216,10 +226,20 @@ public function getShippingAndTaxEstimate( $quote )
      * @param Mage_Sales_Model_Quote $quote    Quote which has been updated to use new shipping rate
      * @param string $shippingRateCode         Shipping rate code composed of {carrier}_{method}
      */
-    public function applyShippingRate($quote, $shippingRateCode) {
+    public function applyShippingRate($quote, $shippingRateCode, $clearTotalsCollectedFlag = true ) {
```

Comment:
where you pass false?

Reply:
In pre-auth order creation, it seems this flag is set to false in the `create_order` method.  Though the code is well tested, I can not logically think of why we would ever do harm by clearing the pre-calculated totals, and it theoretically always seems good to do this.  All other calls use the default true, so, I will remove this.